cernopendata/modules/fixtures/data/records/lhcb-learning-resources.json

```diff
@@ -0,0 +1,32 @@
+[
+{
+  "abstract": {
+    "description": "LHCb is part of the International Masterclass Programme: each year, high-school students reach their nearby universities and become LHCb physicists for a day. Their goal is to select in real LHCb events a particular decay of the D0 meson, where a Kaon and a Pion are produced. On such events, the D0 lifetime is measured, using the same analysis tools and techniques LHCb researchers use every day."
+  }, 
+  "accelerator": "CERN-LHC", 
+  "collaboration": {
+    "name": "LHCb collaboration"
+  }, 
+  "collections": [
+    "LHCb-Learning-Resources"
+  ], 
+  "experiment": "LHCb", 
+  "keywords": [
+    "masterclasses", 
```

Comment:
I have noticed this capitalisation issue in a few places and not only in the `keyword` property: `masterclasses` vs `Masterclasses`. 
Not sure if it's worth fixing, but just noting it down.

Reply:
In the `keywords` it will be good to fix it now, so that we have harmony in such a "controlled vocabulary" field.  Would you prefer mixed case there or all lowercase?

Reply:
I would go for `Masterclasses` to match the format in other similar properties (e.g. `Simulated Data` in `type`)

Reply:
I have found only the following keywords that are not lowercase:

```
External resource
LHCb
ATLAS
ALICE
Masterclass
```

The collaboration names should not really go to the keyword field, but I think we should attack this later, once we have keyword facet visible... 

OK for going all lowercase for all keyword fields?

Reply:
Done.